README.md

```diff
@@ -1,92 +1,122 @@
-# OOP school library
+# Enumerable-project
 
-> In this project, I will start building my school library app. I'm practicing design patterns for building OOP applications.
-## Built With
+## Getting Started
 
-- Ruby
+# 📗 Table of Contents
 
-## Getting Started
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
```

Comment:
- [ ] As I mentioned in my above comment's last line, add always relevant information. Because you don't have any `FAQ` for the user, so that's why remove it too from your `README` file.